packages/ag-charts-enterprise/src/features/error-bar/errorBar.ts

```diff
@@ -19,7 +20,20 @@ const {
 } = _ModuleSupport;
 const { Logger } = _Util;
 
+const XVALUE_ERRORS_ID = 'xValue-errors';
+const YVALUE_ERRORS_ID = 'yValue-errors';
+
+type AnyCartesianSeries = _ModuleSupport.CartesianSeries<any, any>;
```

Comment:
Nit: Could you try `unknown` rather than `any` if these parameter types aren't important for us to know here?

Reply:
`<unknown, unknown>` wouldn't compile, but I've used `<_Scene.Node, _ModuleSupport.CartesianSeriesNodeDatum>` which should work for this type. There doesn't to be much that I can do about the types though (`DataModel`, `ProcessedData`, `Scale`).